src/NuGet.config

```diff
@@ -1,16 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Were the changes to this file intentional? Not that it really matters here, but if they weren't, we need to be careful that they're not picked up once we eventually merge into corefx/master.